src/Optimizely/Config/DatafileProjectConfig.php

```diff
@@ -627,14 +721,15 @@ public function getAttribute($attributeKey)
 
     /**
      * @param $experimentKey string Key for experiment.
-     * @param $variationKey string Key for variation.
+     * @param $variationKey  string Key for variation.
```

Comment:
actually i started my before the pr of duplicate exp key issue was merged with master so i just update my branch with the latest master branch to use that instead of doing that task again